org.mixedrealitytoolkit.spatialmanipulation/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
-## [3.2.0-development] - 2024-03-20
```

Comment:
please add a new section for 3.2.1 and add a bullet point to the added section to describe the change (can look to other package's change logs for example)